### DISCUSSION TITLE
Be able to specify a different Status for PR and Issue in `Item added to project` Workflow

### DISCUSSION THREAD

**Discussion by C0D3-M4513R** [community/community#44895] [Projects and Issues]
### Select Topic Area

Product Feedback

### Body

I would like PR's to be added in a different Status row, than Issues in the `Item added to project`  workflow for Projects.
e.g. 
- When a PR is added to a Project I want to add it to the In-Progress Status in my Project
- When a issue is added to a Project I want to add it to the Todo Status in my Project

**Comment by yuki2006** [comment]
I also want exactly this.

Or, I want to set it so that when a Pull Request is opened, it becomes In Progress.

**Comment by hohwille** [comment]
Workflows of projects can be very helpful to automate things.
However, the implementation of it is taking away a lot for flexibility for no reason.
Workflows like `Item added to project` are hardcoded rules that can be configured flexible:
* When an item is added to the project: I can check `Issue` and/or `pull request`
* Set value: I can select one Status like `New` or `Review`

What I really want is that new `issue` gets `Status` set to `New` and new `pull request` gets `Status` set to `Review`.
IMHO makes totally sense. But does not seem possible.
I can uncheck `pull request` so all works fine for issues but IMHO I cannot duplicate the build-in rule and configure that `pull request` gets into `Review`.

p.s.: It is always good to create a unique language (see e.g. DDD). GitHub is using `workflow` for GitHub-Actions as well as for Projects with quite different semantics causing confusion.

**Comment by hohwille** [comment]
Current UI:
![image](https://github.com/user-attachments/assets/0d7274d2-de94-4d58-b2f9-a13edefb85f6)
Constructive suggestion for improved UI:
![image](https://github.com/user-attachments/assets/8b48d560-bfd1-4ec4-af23-fa1656a05840)
If drop-downs on the right also allow to choose an empty option, you loose no feature but win the requested flexibility.
WDYT?

**Comment by hohwille** [comment]
![image](https://github.com/user-attachments/assets/a4efe182-6868-4d20-a49a-99d06f579650)

**Comment by hohwille** [comment]
BTW: If you want to think beyond, you could even support adding discussions to the board so if someone asks a new question this also gets attention on the board and team can plan to answer the question ;)
